src/graphics.c

```diff
@@ -172,3 +188,92 @@ void graphics_char(int x, int y, char ch, struct graphics_color fgcolor,
     return graphics_bitmap(x, y, FONT_WIDTH, FONT_HEIGHT, &fontdata[u],
                            fgcolor, bgcolor);
 }
+
+uint8_t graphics_verify_color_range(uint8_t x) {
```

Comment:
This function is redundant, an unsigned 8 bit integer can only hold values between 0 and 2255